docs/001_develop/03_client-capabilities/008_interaction/slider.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: 'Slider'
+sidebar_label: 'Slider'
+id: client-interaction-slider
+keywords: [interaction, interactive, input, slider, form, data]
+tags:
+- interaction
+- interactive
+- input
+- slider
+- form
+- data
+sidebar_position: 8
+---
+
+import SliderDemo from '../../../../examples/ui/client-capabilities/interaction/slider.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Slider
+
+Use cases:
+* Interact with data visualization
+* Input for numeric values
+* Dynamic filters
+
+## Example
+
+<SliderDemo />
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+  <TabItem value="genesis">
+
+    Declaration
+    ```html
+    <rapid-slider>
+      <rapid-slider-label position="0"> Low </rapid-slider-label>
+      <rapid-slider-label position="50"> Mid </rapid-slider-label>
+      <rapid-slider-label position="100"> High </rapid-slider-label>
+    </rapid-slider>
+    ```
+
+    Usage
+    ```typescript
+    @customElement({
+      name: 'my-element',
+      template: html`
+            <rapid-slider @change="${(x) => x.handleChange()}"
+                          :value="${sync((x) => x.sliderValue)}">
```

Comment:
can we just use the change event handler instead of sync so the examples are equivalent across frameworks?